packages/components/molecules/f-restaurant-card/src/components/subcomponents/DeliveryTimeMeta/DeliveryTimeMeta.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div>
+        <icon-text
+            v-if="eta"
+            :text="eta"
+            :accessible-text="etaForScreenReaders"
+            :is-bold="true"
+            :color="$options.PieTokens.theme.jet.color.global.green"
+            :hide-icon-in-tile-view="true"
+            :is-list-item="isListItem">
+            <clock-icon />
+        </icon-text>
+
+        <icon-text
+            v-else-if="distance"
+            :text="distance"
+            :accessible-text="distanceForScreenReaders"
+            :is-bold="true"
+            :hide-icon-in-tile-view="true"
+            :is-list-item="isListItem">
+            <map-pin-icon />
+        </icon-text>
+
+        <icon-text
+            v-else-if="address"
+            :text="address"
+            :accessible-text="addressForScreenReaders"
+            :is-list-item="isListItem">
+            <map-pin-icon />
+        </icon-text>
+    </div>
+</template>
+
+<script>
+import { MapPinIcon, ClockIcon } from '@justeat/f-vue-icons';
+import PIE_TOKENS from '@justeat/pie-design-tokens/dist/tokens.json';
+import IconText from '../IconText.vue';
+
+export default {
+    PieTokens: PIE_TOKENS,
```

Comment:
What's the benefit of doing this as opposed to exposing the pie tokens as part of the data object?

 I'm not sure if this approach offers any benefit and it seems a bit non-standard? Really interested to hear your thoughts!

Reply:
yeah, I think that exposing the tokens as a part of the data object might be more readable, so I'm happy to refactor. This approach looks a bit neater, but I guess it's just a personal preference. @dawidchar what do you think?

Reply:
Because the pie tokes don't need to be reactive, this is the preferred approach as it will import them statically instead of reactively